python/idsse_common/idsse/common/aws_utils.py

```diff
@@ -2,56 +2,50 @@
 # -------------------------------------------------------------------------------
 # Created on Tue Feb 14 2023
 #
-# Copyright (c) 2023 Regents of the University of Colorado. All rights reserved.
+# Copyright (c) 2023 Colorado State University. All rights reserved. (1)
+# Copyright (c) 2023 Regents of the University of Colorado. All rights reserved. (2)
 #
 # Contributors:
-#     Geary J Layne
+#     Geary Layne (2)
+#     Paul Hamer (1)
 #
 # -------------------------------------------------------------------------------
 
 import logging
-import fnmatch
 import os
 from collections.abc import Sequence
-from datetime import datetime, timedelta, UTC
 
-from .path_builder import PathBuilder
-from .utils import TimeDelta, datetime_gen, exec_cmd
+from .protocol_utils import ProtocolUtils
+from .utils import exec_cmd
 
 logger = logging.getLogger(__name__)
 
 
-class AwsUtils():
+class AwsUtils(ProtocolUtils):
     """AWS Utility Class"""
 
-    def __init__(self,
-                 basedir: str,
-                 subdir: str,
-                 file_base: str,
-                 file_ext: str) -> None:
-        self.path_builder = PathBuilder(basedir, subdir, file_base, file_ext)
-
-    def get_path(self, issue: datetime, valid: datetime) -> str:
-        """Delegates to instant PathBuilder to get full path given issue and valid
+    def ls(self, path: str, prepend_path: bool = True) -> Sequence[str]:
+        """Execute a 'ls' on the AWS s3 bucket specified by path
 
         Args:
-            issue (datetime): Issue date time
-            valid (datetime): Valid date time
+            path (str): s3 bucket
+            prepend_path (bool): Add to the filename
 
         Returns:
-            str: Absolute path to file or object
+            Sequence[str]: The results sent to stdout from executing a 'ls' on passed path
         """
-        lead = TimeDelta(valid-issue)
-        return self.path_builder.build_path(issue=issue, valid=valid, lead=lead)
+        return self.aws_ls(path, prepend_path)
 
-    def aws_ls(self, path: str, prepend_path: bool = True) -> Sequence[str]:
-        """Execute an 'ls' on the AWS s3 bucket specified by path
+    @staticmethod
```

Comment:
I think it makes more sense to rename 'aws_ls' to just 'ls' vs having 'ls' call 'aws_ls'. Of course it will no longer be a static method